apiary.apib

```diff
@@ -3315,6 +3317,151 @@ Get information about a meeting.
+            + end_date: `2022-05-13` (string)  - Filter on dates that are on or before the given day
+            + relates_to (object, optional) - Filter calls by related object (currently only companies allowed)
+                + id: `415fa7e0-e319-0952-b973-f067f2b49d56` (string)
+                + type: `company` (string)
+            + milestone_id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, optional) - Filter calls by milestone
+            + call_outcome_id: `6bd20dd8-dc3f-0e68-8919-dee0be6efe55` (string, optional) - Filter on done calls by outcome
+        + page (object, optional)
+            + size: `20` (number, optional) - The amount of entries returned per request, max 1 to 100
+                + Default: `20`
+            + number: `1` (number, optional) - The current page, the first page is 1
+                + Default: `1`
+
++ Response 200 (application/json)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string)
+                + added_at: `2021-11-21T18:14:15+00:00` (string, nullable)
+                + customer (object, nullable)
+                    + id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string)
+                    + type: `company` (string)
+                + description: `Call with customer` (string, nullable)
+                + optional_contact (object, nullable)
+                    + id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string)
+                    + type: `contact` (string)
+                + outcome (object, nullable)
+                    + type: `callOutcome` (string)
+                    + id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string)
+                + outcome_summary: `Called, but was not available` (string, nullable)
+                + responsible_user (object, nullable)
+                    + id: `f29abf48-337d-44b4-aad4-585f5277a456` (string)
+                    + type: `user` (string)
+                + scheduled_at: `2021-11-21T19:14:15+00:00` (string)
+                + status: `open` (enum[string])
+                    + Members
+                        + `open`
+                        + `done`
+        + meta (object) - Only included with request parameter `includes=pagination`
+            + page (Pagination)
+            + matches: `12` (number)
+
+### calls.info [POST /calls.info]
+
+Get information about a call.
+
++ Request (application/json)
+
+    + Attributes (object)
+        + id: `6fac0bf0-e803-424e-af67-76863a3d7d16` (string, required)
+
++ Response 200 (application/json)
+
+    + Attributes (object)
+        + data (object)
+            + id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string)
+            + added_at: `2016-02-01T16:44:33+00:00` (string, nullable)
+            + completed_at: `2016-02-04T16:44:33+00:00` (string, nullable)
+            + customer (object, nullable)
```

Comment:
Q: Not everyone you call is already a customer. What about `recipient`?

Reply:
We can't add non-TL customers to the calls so they are customers.

Reply:
I understand but a contact / company is not automatically a customer, it could be a lead or a partner or ...